docs/userguide/modules/creating-service-bindings/pages/custom-path-injection.adoc

```diff
@@ -132,3 +132,77 @@ metadata:
 spec:
     secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581
 ....
+
+== Workload resource mapping
+
+The specification also provides a way of defining exactly where binding data needs to be projected
+into, which may work in situations where the above methods won't work. This is done by defining
+`ClusterWorkloadResourceMapping.servicebindings.io` resources, which define where binding data gets
+projected for a workload type.  This works for both CoreOS and specification service binding
+resources.
+
+The schema of these resources looks like the following:
+[source,yaml]
+....
+apiVersion: servicebinding.io/v1alpha3
+kind: ClusterWorkloadResourceMapping
+metadata:
+  name: # string <1>
+  generation: # int64, defined by the Kubernetes control plane
```

Comment:
Is this `metadata.generation` something that a user needs to include or set for the resource (YAML/JSON) or is managed/generated by Kubernetes/Openshift itself automatically (similarly as `creationTimestamp`)?

In case it is generated, I would not mention it here.

Reply:
It's autogenerated, and the user doesn't have to specify it.  I had it in because the spec mentions it; I've taken it out for now.